package.json

```diff
@@ -36,11 +48,11 @@
     "SECURITY.md"
   ],
   "dependencies": {
-    "axios": "^1.4.0",
-    "typescript": "^5.1.3"
+    "typescript": "^5.3.3",
+    "zod": "^3.22.4"
```

Comment:
Did the optional peer dependency trick not work out? (Happy to help get that working!)